### ISSUE TITLE
Addition of Detailed Docstrings

### ISSUE THREAD

**Issue by syedhamidali** [MaceKuailv/seaduck#106]
I would like to suggest an enhancement to the `seaduck` package documentation, specifically regarding the addition of comprehensive docstrings for several key functions. The functions in question are listed in the [API Reference section](https://macekuailv.github.io/seaduck/api_reference/apiref_OceData.html) of the `seaduck` documentation.

As an example, for the function `def find_rel_t_lin(t):`, an expanded docstring could include:

- A brief description of the function's purpose and operation.
- Detailed descriptions of its parameters and return values.
- Possible exceptions or edge cases to consider.

Here's a suggested template for the `find_rel_t_lin` function:

```python
def find_rel_t_lin(t):
    """
    Find the relative coordinates based on the temporal direction using a 2-point linear scheme.

    Parameters
    ----------
    t : [type]
        Description of parameter `t`, including its role and any expected value range or type.

    Returns
    -------
    [return type]
        Description of the return value, including what it represents and how it should be interpreted.

    Raises
    ------
    [ExceptionType]
        Description of any exceptions that might be raised by this function and under what conditions.

    Notes
    -----
    Any additional notes, usage examples, or references to related functions.

    """
```

Similarly, detailed docstrings for the following functions would enhance the usability and accessibility of the `seaduck` package:

- `find_rel_v(z)`
- `find_rel_v_lin(z)`
- `find_rel_vl(z)`
- `find_rel_vl_lin(z)`
- `show_alias()`

This enhancement request is part of the ongoing JOSS review (https://github.com/openjournals/joss-reviews/issues/5967).

**Comment by MaceKuailv** [comment]
Good catch. 

The methods you mentioned except for show_alias are considered internal methods that are not intended for the users. Pål has also pointed this out, but I didn't make the separation carefully enough. This will be solved in the next commit.

**Comment by MaceKuailv** [comment]
closed with PR #99
